app/models/moab_storage_root.rb

```diff
@@ -4,11 +4,16 @@
 # Metadata about a Moab storage root (a POSIX file system which contains Moab objects).
 class MoabStorageRoot < ApplicationRecord
   has_many :complete_moabs, dependent: :restrict_with_exception
+  has_many :preserved_objects, through: :complete_moabs
```

Comment:
this ActiveRecord association is now a remnant of an earlier implementation.  i think i'd find it useful when doing investigations from the console (poking around e.g. stage or prod DB).  but since it was introduced in this PR but isn't actually called by any of the code in it, i'd be happy to get rid of it.